include/irr/builtin/glsl/bxdf/brdf/specular/blinn_phong.glsl

```diff
@@ -3,25 +3,59 @@
 
 #include <irr/builtin/glsl/bxdf/common.glsl>
 #include <irr/builtin/glsl/bxdf/common_samples.glsl>
+#include <irr/builtin/glsl/bxdf/brdf/specular/ndf/blinn_phong.glsl>
 
-float irr_glsl_blinn_phong(in float NdotH, in float n)
+float irr_glsl_blinn_phong_rdf(in float NdotH, in float n)
```

Comment:
NDF

Reply:
nah, ndf is in `bxdf/brdf/ndf/blinn_phong.glsl`. This is ndf with some shadowing term incorporated, generally this model is weird and has lots of definitions... 
http://renderwonk.com/publications/s2010-shading-course/hoffman/s2010_physically_based_shading_hoffman_b.pdf
http://www.thetenthplanet.de/archives/255

Reply:
interesting